scripts/install/install-apt-packages.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -ex
+
+# Installs APT packages (e.g. debian/ubuntu packages installed for all users).
+# Requires being logged in as root or the ability to successfully execute sudo.
+
+THIS_DIR="$(dirname "$(readlink -f "${0}")")"
+# shellcheck source=/var/panoptes/POCS/scripts/install/install-helper-functions.sh
+source "${THIS_DIR}/install-helper-functions.sh"
+
+# Suppress prompting for input during package processing, unless DEBIAN_FRONTEND
+# is already set. The tzdata 
+if [ -z "${DEBIAN_FRONTEND}" ] ; then
+  export DEBIAN_FRONTEND=noninteractive
+fi
+
+# Generate the basic apt-get install command, minus the list of packages.
+declare -a apt_get_install=(apt-get install --no-install-recommends --yes)
+# shellcheck disable=SC2119
+apt_proxy_url="$(get_apt_proxy_url)"
+if [ -n "${apt_proxy_url}" ] ; then
+  apt_get_install+=(-o "Acquire::HTTP::Proxy=${apt_proxy_url}")
```

Comment:
Should we use the HTTPS by default?

Reply:
That would require adding a trusted cert to the proxy setup. This is only for the communication between apt-get and the proxy... IIUC. I'm hoping that the communication thereafter is over TLS, but don't know.